model/task_lifecycle_test.go

```diff
@@ -1190,6 +1190,66 @@ func TestAbortTask(t *testing.T) {
 	})
 
 }
+func TestTryDequeueAndAbortCommitQueueVersionWithBlacklist(t *testing.T) {
+	assert.NoError(t, db.ClearCollections(patch.Collection, VersionCollection, task.Collection, build.Collection, commitqueue.Collection))
+	patchID := "aabbccddeeff001122334455"
```

Comment:
FYI you can just do mgobson.NewObjectId().Hex()

Reply:
I guess I should read earlier comments...